### PR TITLE
Completed Exercise 1

### PR DESCRIPTION
**Pull request by tusharm-cuelogic**
# Understands Following things in Exercise 1
# How to create .py file.
# How to execute first program like "Hello Word!!!".
# How to run python file on terminal.
# How errors will display.
# Use '#' to comment out single line.

### CODE REVIEW

File: ex1.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+print "I'd much rather you 'not'."
+print 'I "said" do not touch this.'
+
+
+# Understands Following things in Exercise 1
+# How to create .py file.
+# How to execute first program like "Hello Word!!!".
+# How to run python file on terminal.
+# How errors will display.
+# Use '#' to commentout single line.
```
Comment (onkartibe) [inline ex1.py:15]: add line at every end of the file.

File: ex1.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+print "I like typing this."
+print "This is fun."
+print 'Yay! Printing.'
+print "I'd much rather you 'not'."
+print 'I "said" do not touch this.'
+
+
+# Understands Following things in Exercise 1
+# How to create .py file.
+# How to execute first program like "Hello Word!!!".
```
Comment (onkartibe) [inline ex1.py:12]: can you tell me how to execute the program?

**Comment by onkartibe** [comment]
@tusharm-cuelogic you have set targeted commit to master branch. It should be dev branch. Close the PR and generate new one with target branch to Dev.

File: ex1.py (hunk 2 above)
Comment (tusharm-cuelogic) [inline ex1.py:12]: to execute program 
python ex1.py

**Pull request closed without merging**